patches/net/minecraft/client/gui/components/AbstractWidget.java.patch

```diff
@@ -9,6 +9,48 @@
      private static final double PERIOD_PER_SCROLLED_PIXEL = 0.5;
      private static final double MIN_SCROLL_PERIOD = 3.0;
      protected int width;
+@@ -91,13 +_,25 @@
+     protected abstract void renderWidget(GuiGraphics p_282139_, int p_268034_, int p_268009_, float p_268085_);
+ 
+     public static void renderScrollingString(
++            GuiGraphics p_281620_, Font p_282651_, Component p_281467_, int p_283621_, int p_282084_, int p_283398_, int p_281938_, int p_283471_, boolean dropShadow
```

Comment:
More of a nitpick, but can you re-order these overloads like this:
- Original 8-param method
- 8-param method with added shadow parameter
- Original 9-param method
- 9-param method with added shadow parameter

This should make the patch a bit easier to read